src/pages/Products/PC/components/FormProps.tsx

```diff
@@ -0,0 +1,34 @@
+import { TPCNewOrUpdated } from "../../../../types/PC/TPC";
+import * as Yup from "yup";
+
+export const initialValues: TPCNewOrUpdated = {
+    processorBrand: "",
+    processorName: "",
+    graphicsCardBrand: "",
+    graphicsCardName: "",
+    ramCapacity: "",
+    driveCapacity: "",
+    driveType: "",
+    operatingSystem: "",
+    price: 0,
+};
+
+export const validationSchema = Yup.object().shape({
+    processorBrand: Yup.string().required("Processor brand is required"),
```

Comment:
Jak chcesz sprawdzić response z backendu dla null to możesz zakomentować to, lub którykolwiek atrybut.